app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java

```diff
@@ -119,7 +104,7 @@ public void onLoadResource(WebView view, String url) {
     }
 
     @Override
-    public WebResourceResponse shouldInterceptRequest(WebView view, WebResourceRequest request) {
+    public WebResourceResponse shouldInterceptRequest(WebView view, final WebResourceRequest request) {
```

Comment:
\[PMD\] INFO: The method 'shouldInterceptRequest' has a Cyclomatic Complexity of 6\.